### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by flands**
Attempt to address #480 and addresses #347

* Add a table of contents
* Improve layout through the use of tables
* Ensure updated SIG information
* Remove duplicated information and leverage linking

### CODE REVIEW

**Comment by flands** [comment]
* Looking for feedback on proposal
* TODO: adjust formatting on tables a bit
* Also considered using badges instead of links in SIG section

**Review by tigrannajaryan — APPROVED** [review]
Very nice.

File: README.md [hunk 1]
```diff
@@ -2,9 +2,32 @@
...
+<summary>
+Table of Contents
+</summary>
+
+* [Getting Involved](#getting-involved)
+* [Governing Bodies](#governing-bodies)
+* [Communication](#communication)
+  * [Calendar](#calendar)
+  * [Mailing Lists](#mailing-lists)
+  * [License](#license)
```
Comment (SergeyKanzhelev) [inline README.md:15]: not under communications I think

File: README.md [hunk 2]
```diff
@@ -2,9 +2,32 @@
...
+  * [Logos and Branding Guide](#logo-and-branding-guide)
+* [Community Meetings](#community-meetings)
```
Comment (SergeyKanzhelev) [inline README.md:17]: this may be under communications

File: README.md [hunk 3]
```diff
@@ -2,9 +2,32 @@
 
 OpenTelemetry community content.
```
Comment (SergeyKanzhelev) [inline README.md:3]: small logo here? and link to website?

File: README.md [hunk 4]
```diff
@@ -14,264 +37,91 @@ There is a lot to do! If you are interested in getting involved, please join the
...
+as CNCF IP Policy
+[dictates](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy).
+This [blog post](https://www.cncf.io/blog/2017/02/01/cncf-recommends-aslv2/)
+explains the reasoning behind choosing this license.
 
 ### Logos and Brand Guide
 
-The OpenTelemetry logos and brand guide can be found in the [CNCF artwork repository](https://github.com/cncf/artwork/tree/master/projects/opentelemetry).
+The OpenTelemetry logos and brand guide can be found in the [CNCF artwork
+repository](https://github.com/cncf/artwork/tree/master/projects/opentelemetry).
```
Comment (SergeyKanzhelev) [inline README.md:82]: take this chance to plug your presentation template?

File: README.md (hunk 3 above)
Comment (SergeyKanzhelev) [reply README.md:3]: also maybe make  "OpenTelemetry community content." an H1 instead of the word "Community"?

**Comment by SergeyKanzhelev** [comment]
left a few suggestions. Happy to merge as is, no objections

**Comment by flands** [comment]
@SergeyKanzhelev incorporated feedback -- ready for merge

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**